### ISSUE TITLE
Deleting single tag is delteing all of them

### ISSUE THREAD

**Issue by tylkotroszke** [klausmeyer/docker-registry-browser#204]
For example when I have few tags for one image and I want to delete one of them, I type the tag and then press "delete tag" button. The feature is deleting all tags for choosen image, not the one i choose.

**Comment by klausmeyer** [comment]
Hi @tylkotroszke,

thanks for reporting this bug. I was able to reproduce it by the following steps:

1. Create a few docker images locally with the same content but different tags:
```
$ docker pull busybox

$ docker tag busybox localhost:5000/some/image:tag1
$ docker tag busybox localhost:5000/some/image:tag2
$ docker tag busybox localhost:5000/some/image:tag3

$ docker push localhost:5000/some/image:tag1
$ docker push localhost:5000/some/image:tag2
$ docker push localhost:5000/some/image:tag3
```
2. Opening the Docker Registry Browser and delete the tag "tag1"
3. Tags "tag2" and "tag3" are also gone

When I tried the same with tags having different contents I was not able to reproduce.

After a bit of research I've learned that there is currently no API to delete a single tag in the registry.

The endpoint I'm using is designed to delete the image referenced by the tag and not the tag reference itself. So basically this issue could only be solved by a new endpoint in the registry itself.

Source: https://serverfault.com/a/872336

For now the only thing I could do is to change the warning text and mention that deleting the tag will also delete also delete all tags referencing the same image-version.

What do you think?

Best, Klaus

**Comment by tylkotroszke** [comment]
I see you are right we were checking that on image with same content. It would be nice to have such a information when trying to delete tag. It was really confusing when all tags were deleted that why we created issue :)

**Comment by klausmeyer** [comment]
> It would be nice to have such a information when trying to delete tag.

:+1: Sure will add it.

**Comment by klausmeyer** [comment]
I've added the warning message now. Will be available as `v1.2.2` tag or in `latest` soon.

**Comment by u1735067** [comment]
Hi, this seems to be supported now (https://serverfault.com/questions/872332/easy-way-to-remove-a-tag-from-a-private-docker-registry/872336#872336 / https://github.com/opencontainers/distribution-spec/blob/v1.0/spec.md#deleting-tags), could this be implemented?

**Comment by klausmeyer** [comment]
@u1735067 I just had a look and the feature isn't yet available in the latest release of the registry (2.8.1). They don't seem to add new features in 2.x. Let's wait for 3.0 then.
